src/main/resources/db/changelog/changelog1.sql

```diff
@@ -0,0 +1,11 @@
+-- liquibase formatted sql
+
+-- changeset nika.avalishvili:1
+CREATE TABLE employees (employee_id INT PRIMARY KEY,
+                        first_name VARCHAR(255),
+                        last_name VARCHAR(255),
+                        department VARCHAR(255),
+                        positions VARCHAR(255),
+                        email VARCHAR(255),
+                        employee_status Boolean,
+                        pf_status Boolean)
```

Comment:
Let's rename these fields according to entity changes

Let's rename whole file to contain date of this file creation